tests/test_auth.py

```diff
@@ -0,0 +1,59 @@
+"""Tests for User Authentication"""
+import unittest
+import json
+from app import db, make_app
+
+class TestAuth(unittest.TestCase):
+    """"Testcase for blueprint for authentication"""
+
+    def setUp(self):
+        self.app = make_app(config_name="testing")
+        self.client = self.app.test_client
+        self.user_details = {'email':'someone@gmail.com',
+                             'password':'testing_p@ssword'
+                             }
+        self.unathorized_user_details = {'email': 'uuser@unauthorized.com',
+                                         'password': 'none_provided'
+                                    }
+
+        with self.app.app_context():
+            db.session.close()
+            db.drop_all()
+            db.create_all()
+
+    def testResisterUser(self):
+        """"Method to test a successful registration"""
+        user_register = self.client().post('/flask_api/v1/auth/register', data = self.user_details)
```

Comment:
`/flask_api`? Is it not better if the URI would have the name of the app instead eg `/yummy`?